client/src/controllers/FormsetController.ts

```diff
@@ -224,6 +285,50 @@ export class FormsetController extends Controller<HTMLElement> {
     });
   }
 
+  /**
+   * If the orderInputTargets are present, update the value of each input
+   * to match the current order of the child elements.
+   */
+  updateOrdering() {
```

Comment:
Just a thought, I know we discussed this, but maybe to avoid confusion we should call this method `syncOrder` or `syncOrderInputs` or `syncOrdering`.

This way we can avoid future confusion where developers think this is updating the ordering from a user perspective, instead it's just syncing the DOM order (after any move) with the order input values in each child.

```suggestion
  syncOrdering() {
```